incubator/jaeger/values.yaml

```diff
@@ -0,0 +1,79 @@
+# Default values for jaeger.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cassandra:
+  image:
+    tag: 3.11
+  persistence:
+    enabled: false
+  config:
+    cluster_name: jaeger
```

Comment:
Camel case, please.

Reply:
I am overwriting values in the Cassandra chart and that is how they appear. What's the best way to resolve this?

Reply:
@unguiculus requesting feedback for the above question

Reply:
I can always submit a PR to fix their camel case issue.